massgov/mayflower
# Patternlab/DP-4562  set focus state for search on mobile

## Description
Adds focus to search form on button click in mobile 

## Related Issue / Ticket

- [DP-4526](https://jira.mass.gov/browse/DP-4526)

## Steps to Test

1. From any page in a browser with width less than 620px, click the search button. Focus should be in the search inout box when the mobile menu opens

## Screenshots
![search](https://user-images.githubusercontent.com/18662734/52749469-e8ea6080-2fae-11e9-92b1-db1f41ecb9b6.gif)
